bin/rose-check-software

```diff
@@ -311,8 +306,6 @@ def main(check=check):
 def docs(check=check):
     """Check software dependencies for the documentation builder."""
     ret = check_all('Documentation Builder', [
-        check('python2', (2, 7), (3,), version_template=r'Python (.*)',
-               outfile=2),
         check('graphviz-dot <dot>', command_template='-V',
```

Comment:
Honestly not sure - I think that @oliver-sanders  might be better placed to answer this one!

Reply:
This line refers to the Python version required by the documentation builder, the other line refers to the Python version required by Rose.

Once the docs have been upgraded so that they build under Python 3 we can remove this line entirely.